src/main.rs

```diff
@@ -1,8 +1,15 @@
-use std::path::Path;
+use std::path::PathBuf;
 
 fn main() {
     let mut args = sylt::Args::parse_args_default_or_exit();
-    args.file = Some(Path::new("game.sy").to_path_buf());
+    args.file = Some(PathBuf::from(
+        std::env::args()
+        .nth(1)
```

Comment:
Maybe find the first argument ending in '.sy'? I think I usually write the flags first - and it gets frustrating if it overwrites it - so just swap this out for a `find`?